### PR TITLE
Remove file with same name and different case

### PR DESCRIPTION
**Pull request by alexspeller**
## Description

#6027 includes two files named the same but with different case. This breaks git on case-insensitive filesystems (like MacOS by default) - @drashna  @noroadsleft mentioning you as you were reviewers on that PR

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tada68/keymaps/hhkb68/README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # HHKB68
```
Comment (noroadsleft) [inline keyboards/tada68/keymaps/hhkb68/README.md]: This file should be renamed to fully lowercase `readme.md`. We are trying to enforce lowercase filenames wherever possible, but you are correct that this was missed during review.

**Comment by drashna** [comment]
Sorry, I missed that. 

And yeah, we prefer lowercase names, as it's less prone to "odd issues".

**Comment by alexspeller** [comment]
Makes sense, I renamed to lowercase

**Comment by hillam** [comment]
Whoops! Thanks for catching that.

**Pull request merged**